api_tests/preprints/views/test_preprint_list_mixin.py

```diff
@@ -0,0 +1,156 @@
+from nose.tools import *  # flake8: noqa
+
+from api.base.settings.defaults import API_BASE
+
+from website.util import permissions
+
+from osf_tests.factories import (
+    ProjectFactory,
+    PreprintFactory,
+    AuthUserFactory,
+    SubjectFactory,
+)
+
+from api_tests import utils as test_utils
+
+class PreprintIsPublishedListMixin(object):
+
+    def setUp(self):
+        super(PreprintIsPublishedListMixin, self).setUp()
```

Comment:
@sloria I had a note on the previous version of this ticket commenting on that -- none of the other api tests have been ported and it drew from that functionality heavily. I figured it would be faster to port everything over at once when I understand pytest better than to do one-off tests now.